gnuradio-runtime/apps/gnuradio-config-info.cc

```diff
@@ -48,9 +46,8 @@ int main(int argc, char** argv)
         po::store(po::parse_command_line(argc, argv, desc), vm);
         po::notify(vm);
     } catch (po::error& error) {
-        gr::logger_ptr logger, debug_logger;
-        gr::configure_default_loggers(logger, debug_logger, "gnuradio-config-info.cc");
-        GR_LOG_ERROR(logger, boost::format("ERROR %s %s") % error.what() % desc);
+        std::cerr << "Error: " << error.what() << std::endl << std::endl;
+        std::cerr << desc << std::endl;
```

Comment:
This code had been switched to use the logging framework in #3339, but for this simple command-line tool I think it makes more sense to print the error message (and usage information) directly to stderr.